### PR TITLE
Wip separate cdc

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: mysql/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ARG EVENTUATE_TRAM_VERSION
-FROM eventuateio/eventuate-tram-mysql:$EVENTUATE_TRAM_VERSION
+ARG EVENTUATE_TRAM_IMAGE_VERSION
+FROM test-eventuate-mysql
```
Comment (cer) [inline mysql/Dockerfile]: Do you need a private image version?

File: gradle.properties [hunk 2]
```diff
@@ -5,10 +5,13 @@ bintrayRepoType=defineMe
...
-springBootVersion=2.0.3.RELEASE
-eventuateClientVersion=0.20.1.RELEASE
-eventuateTramVersion=0.20.0.RELEASE
-eventuateLocalVersion=0.30.0.RELEASE
+springBootVersion=2.1.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
+eventuateTramVersion=0.20.0.REDIS-SNAPSHOT
+eventuateLocalVersion=0.15.0-SNAPSHOT
+#eventuateTramImageVersion=$eventuateTramVersion
+eventuateTramImageVersion=0.21.1.RELEASE
```
Comment (cer) [inline gradle.properties]: Aren't there other image versions now?

File: gradle.properties [hunk 3]
```diff
@@ -5,10 +5,13 @@ bintrayRepoType=defineMe
 
 eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,https://dl.bintray.com/eventuateio-oss/eventuate-maven-rc/,file:///Users/cer/.m2/testdeploy
 
-springBootVersion=2.0.3.RELEASE
-eventuateClientVersion=0.20.1.RELEASE
-eventuateTramVersion=0.20.0.RELEASE
-eventuateLocalVersion=0.30.0.RELEASE
+springBootVersion=2.1.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
+eventuateTramVersion=0.20.0.REDIS-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: I don't think REDIS-SNAPSHOT is the current one?

File: mysql/Dockerfile (hunk 1 above)
Comment (dartartem) [reply mysql/Dockerfile]: I will check

File: gradle.properties (hunk 2 above)
Comment (dartartem) [reply gradle.properties]: Yes, it should be eventuate-cdc image. 
And I will use released version.

File: gradle.properties (hunk 3 above)
Comment (dartartem) [reply gradle.properties]: It seems now it is 0.21.0-SNAPSHOT. I will merge latest changes to tram.

**Pull request merged**